sdk/src/main/java/io/customer/sdk/queue/Queue.kt

```diff
@@ -33,7 +41,9 @@ class Queue internal constructor(
         if (isManyTasksInQueue) {
             logger.info("queue met criteria to run automatically")
 
-            // start running queue
+            CoroutineScope(Dispatchers.IO).launch {
```

Comment:
I agree. That's probably what will need to happen. 

I have not yet written tests for the `Queue` class which is why the code is like it is today. I didn't want to try to solve a problem I did not yet have to avoid solving it the wrong way. I am writing tests for `Queue` in this milestone 4 out of 4 right now so this will be addressed then.